src/backend/WorkflowManagement/WorkflowQueueMessage.cs

```diff
@@ -0,0 +1,11 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace WorkflowManagement
+{
+    public class WorkflowQueueMessage
```

Comment:
This is fine for now, but it would be nice to set up the workflow so that we use immutable objects (we could then switch to something like `readonly struct`).

Reply:
Easy enough change to do now - done. :)